### PR TITLE
Fix ASI after a colon

### PR DESCRIPTION
**Pull request by fatfisz**
There is a regression compared to 0.5.7, where this worked correctly:

```css
html {
  background-image:
    linear-gradient(0deg, rgba(255, 255, 255, 0.8), rgba(255, 255, 255, 0.8)),
    url(/static/background.svg);
}
```

One of the newer versions of styled-jsx broke this - a semicolon is added after `background-image:`.

This is fixed in stylis 3.0.17. This branch updates stylis and adds a test to prevent regressions.

### CODE REVIEW

**Comment by fatfisz** [comment]
Related PR in stylis: https://github.com/thysultan/stylis.js/pull/33

File: package.json [hunk 1]
```diff
@@ -62,7 +62,7 @@
     "escape-string-regexp": "1.0.5",
     "source-map": "0.5.6",
     "string-hash": "1.1.1",
-    "stylis": "3.0.15"
+    "stylis": "^3.0.17"
```
Comment (giuseppeg) [inline package.json]: please pin this version, i.e. remove the `^`

File: yarn.lock [hunk 2]
```diff
@@ -4625,9 +4625,9 @@ strip-json-comments@~2.0.1:
   version "2.0.1"
   resolved "https://registry.yarnpkg.com/strip-json-comments/-/strip-json-comments-2.0.1.tgz#3c531942e908c2697c0ec344858c286c7ca0a60a"
 
-stylis@3.0.15:
-  version "3.0.15"
-  resolved "https://registry.yarnpkg.com/stylis/-/stylis-3.0.15.tgz#e0f9bf403048511043b479d39422c835141e0fbb"
+stylis@^3.0.17:
```
Comment (giuseppeg) [inline yarn.lock]: you should update this too

Comment (fatfisz) [inline yarn.lock]: Sorry, that's the limit of what I can do on my phone 😉 I'll take care of this later when I'm in front​ if the PC.

Comment (giuseppeg) [inline yarn.lock]: hah, ok no worries. Thank you!

**Comment by fatfisz** [comment]
I installed manually and updated the `yarn.lock` file based on that.

**Comment by giuseppeg** [comment]
@fatfisz thank you for helping out!! I'll release today or tomorrow - I need to squeeze in another fix.

**Pull request merged**